packages/editor-ui/src/components/NodeCreator/CategorizedItems.vue

```diff
@@ -69,16 +65,25 @@ export default mixins(externalHooks).extend({
 	components: {
 		ItemIterator,
 		NoResults,
-		SlideTransition,
 		SearchBar,
-		TriggerHelperPanel,
 	},
-	props: [
-		'searchItems',
-		'excludedCategories',
-		'excludedSubcategories',
-		'selectedSubcategory',
-	],
+	props: {
+		searchItems: {
+			type: Array as PropType<INodeCreateElement[]>,
+		},
+		excludedCategories: {
+			type: Array,
+			default: () => [],
+		},
+		excludedSubcategories: {
+			type: Array,
+			default: () => [],
+		},
+		selectedSubcategory: {
+			type: Array,
+			default: () => [],
+		},
```

Comment:
for arrays would be more useful to specify the exact type `Array as Whatever[]` since they would default to `any[]`